components/mediation/tasks/org.wso2.micro.integrator.ntask.core/src/main/java/org/wso2/micro/integrator/ntask/coordination/task/store/connector/RDMBSConnector.java

```diff
@@ -181,6 +183,36 @@ public void deleteTasks(String nodeId) throws TaskCoordinationException {
         }
     }
 
+    /**
+     * Deactivates the task.
+     *
+     * @param name - Name of the task.
+     */
+    public void deactivateTask(String name) throws TaskCoordinationException {
+        try (Connection connection = getConnection(); PreparedStatement preparedStatement = connection.prepareStatement(
+                DEACTIVATE_TASK)) {
```

Comment:
I prefer the name UPDATE_TASK_STATUS_TO_DEACTIVATED since it describes to query better. WDYT?